flink-libraries/flink-table/src/test/java/org/apache/flink/table/api/java/stream/sql/ProcTimeRowStreamAggregationSqlITCase.java

```diff
@@ -0,0 +1,317 @@
+		expected.add("4,6");
+		expected.add("4,6");
+		expected.add("4,7");
+		expected.add("4,8");
+		expected.add("5,10");
+		expected.add("5,10");
+		expected.add("5,11");
+		expected.add("5,12");
+		expected.add("5,13");
+
+		StreamITCase.compareWithList(expected);
+	}
+	
+	@Test
+	public void testSumAggregatation() throws Exception {
+		StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+		StreamTableEnvironment tableEnv = TableEnvironment.getTableEnvironment(env);
+		StreamITCase.clear();
+
+		env.setParallelism(1);
+		
+		DataStream<Tuple5<Integer, Long, Integer, String, Long>> ds = StreamTestData.get5TupleDataStream(env);
+		Table in = tableEnv.fromDataStream(ds, "a,b,c,d,e");
+		tableEnv.registerTable("MyTable", in);
+
+		String sqlQuery = "SELECT a, SUM(c) OVER (PARTITION BY a ORDER BY procTime() ROWS BETWEEN 2 PRECEDING AND CURRENT ROW) AS sumC FROM MyTable";
+		Table result = tableEnv.sql(sqlQuery);
+
+		DataStream<Row> resultSet = tableEnv.toDataStream(result, Row.class);
+		resultSet.addSink(new StreamITCase.StringSink());
+		env.execute();
+
+		List<String> expected = new ArrayList<>();
+		expected.add("1,0");
+		expected.add("2,1");
+		expected.add("2,3");
+		expected.add("3,3");
+		expected.add("3,7");
+		expected.add("3,9");
+		expected.add("4,6");
+		expected.add("4,13");
+		expected.add("4,15");
+		expected.add("4,17");
+		expected.add("5,10");
+		expected.add("5,21");
+		expected.add("5,23");
+		expected.add("5,26");
+		expected.add("5,27");
+
+		StreamITCase.compareWithList(expected);
+	}
+	
+	@Test
+	public void testSumMinAggregatation() throws Exception {
+		StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+		StreamTableEnvironment tableEnv = TableEnvironment.getTableEnvironment(env);
+		StreamITCase.clear();
+
+		env.setParallelism(1);
+		
+		DataStream<Tuple5<Integer, Long, Integer, String, Long>> ds = StreamTestData.get5TupleDataStream(env);
+		Table in = tableEnv.fromDataStream(ds, "a,b,c,d,e");
+		tableEnv.registerTable("MyTable", in);
+
+		String sqlQuery = "SELECT a, "
+				+ "SUM(c) OVER (PARTITION BY a ORDER BY procTime() ROWS BETWEEN 2 PRECEDING AND CURRENT ROW) AS sumC,"
+				+ "MIN(c) OVER (PARTITION BY a ORDER BY procTime() ROWS BETWEEN 2 PRECEDING AND CURRENT ROW) AS minC"
+				+ " FROM MyTable";
+		Table result = tableEnv.sql(sqlQuery);
+
+		DataStream<Row> resultSet = tableEnv.toDataStream(result, Row.class);
+		resultSet.addSink(new StreamITCase.StringSink());
+		env.execute();
+
+		List<String> expected = new ArrayList<>();
+		expected.add("1,0,0"); 
+		expected.add("2,1,1"); 
+		expected.add("2,3,1"); 
+		expected.add("3,3,3"); 
+		expected.add("3,7,3"); 
+		expected.add("3,9,4"); 
+		expected.add("4,6,6"); 
+		expected.add("4,13,6"); 
+		expected.add("4,15,7"); 
+		expected.add("4,17,8"); 
+		expected.add("5,10,10");
+		expected.add("5,21,10");
+		expected.add("5,23,11");
+		expected.add("5,26,12");
+		expected.add("5,27,13");
```

Comment:
This result incorrect.The reason for the error I left the comments in `DataStreamOverAggregate.scala`